src/app/work-item/work-item-board/work-item-board.module.ts

```diff
@@ -3,17 +3,18 @@ import { CommonModule }       from '@angular/common';
 import { CountPipe } from 'ng2bln-count-pipe';
 
 import { AlmIconModule } from './../../shared-component/icon/almicon.module';
-import { WorkItemBoardComponent }   from './work-item-board.component';
 import { BoardRoutingModule }   from './work-item-board-routing.module';
+import { WorkItemBoardComponent }   from './work-item-board.component';
+import { WorkItemQuickAddModule } from './../work-item-quick-add/work-item-quick-add.module';
 //pipes
 
 import { AlmFilterBoardList } from './../../pipes/alm-board-filter.pipe';
 
 
 
 @NgModule({
-  imports:      [ AlmIconModule, BoardRoutingModule, CommonModule],
-  declarations: [ AlmFilterBoardList, CountPipe , WorkItemBoardComponent, ],
+  imports:      [ AlmIconModule, BoardRoutingModule, CommonModule , WorkItemQuickAddModule],
```

Comment:
Just for readability and better coding style. A single should not cross the max limit of 80 characters in a line.